.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,7 +12,7 @@ tests/ci/cancel_and_rerun_workflow_lambda/app.py
 - Backward Incompatible Change
 - Build/Testing/Packaging Improvement
 - Documentation (changelog entry is not required)
-- Critical Bug Fix (crash, LOGICAL_ERROR, data loss, RBAC)
```

Comment:
Why?

Reply:
Ah, I see: https://github.com/ClickHouse/ClickHouse/pull/71679

Reply:
I still don't see why. This was introduced to make sure some PRs were always backported (the ones related to logical errors and crashes). Now we'll go back to missing backports

Reply:
But we still need to backport the fixes. We can change the category to call it `Critical bugfixes and logical errors` if that's better